indico/modules/users/templates/_category.html

```diff
@@ -16,55 +16,59 @@
     </span>
 {% endmacro %}
 
-{% macro _category_item(category, truncated_path, extra_info='') %}
-    <li class="category-box">
-        <div>
-            <span class="category-box-info">
-                <a href="{{ category.url }}">{{ category.title }}</a>
-                {{ _category_path(category, truncated_path) }}
-                {{ caller() }}
-            {# The caller must provide the closing span tag (</span>). #}
-            {# TODO This should be changed once Jinja is updated to 2.8 with support for set block. #}
+{% macro _category_item(category, truncated_path, extra_info) %}
```

Comment:
`extra_info=''` so we don't rely on `undefined` evaluating as an empty string in cases where we don't provide this argument (i still want to enable `StrictUndefined` at some point, at least when in debug mode)